core/domain/stats_jobs_continuous.py

```diff
@@ -522,29 +522,86 @@ def entity_classes_to_map_over(cls):
 
     @staticmethod
     def map(item):
-        if InteractionAnswerSummariesMRJobManager._entity_created_before_job_queued( # pylint: disable=line-too-long
-                item):
+        def _encode(item):
+            """Encodes a Python value as UTF-8."""
+            if isinstance(item, basestring):
+                return item.encode('utf-8')
+            elif isinstance(item, int) or isinstance(item, float):
+                return item
+            elif isinstance(item, list):
+                return [_encode(subitem) for subitem in item]
+            elif isinstance(item, dict):
+                return {
+                    key: _encode(value) for key, value in item.iteritems()
+                }
+            else:
+                raise Exception('Cannot encode item: %s' % item)
 
-            # All visualizations desired for the interaction.
-            visualizations = (
-                interaction_registry.Registry.get_interaction_by_id(
-                    item.interaction_id).answer_visualizations)
+        # pylint: disable=line-too-long
```

Comment:
Think you need to put this to the right of the next line, otherwise it will apply to everything below (not just the next line). It's OK if it causes the line to be > 80 chars.